mock/mock.go

```diff
@@ -810,7 +810,7 @@ func (f *FunctionalOptionsArgument) String() string {
 // and the values to check of
```

Comment:
This sentence looks unfinished. It needs at last a dot, but I wonder if some word is missing.

Disclaimer: I'm not a native English speaker.

Reply:
what about: 
`FunctionalOptions returns an FunctionalOptionsArgument object containing the expected functional-option to check for`

im trying to keep it similar to
Lines 782 and 765

Do you think it need more detailed description?